Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/General/OutlookGridDefaultGroup.cs

```diff
@@ -229,7 +229,7 @@ public virtual string Text
         /// <summary>
         /// Gets or sets the associated DataGridView column.
         /// </summary>
-        public virtual OutlookGridColumn Column { get => _column; set => _column = value; }
+        public virtual OutlookGridColumn Column { get => _column!; set => _column = value; }
```

Comment:
specify `DissallowNull` to prevent passing null into here, due to ignoring it on the return !

Reply:
Changed